frontend/talentsearch/src/js/components/createAccount/CreateAccountPage.tsx

```diff
@@ -0,0 +1,329 @@
+import * as React from "react";
+import { imageUrl, navigate } from "@common/helpers/router";
+import { useIntl } from "react-intl";
+import { Alert } from "@common/components";
+import { BellIcon } from "@heroicons/react/outline";
+import { FormProvider, useForm } from "react-hook-form";
+import { toast } from "react-toastify";
+import { Input, RadioGroup, Submit } from "@common/components/form";
+import { commonMessages, errorMessages } from "@common/messages";
+import { enumToOptions } from "@common/helpers/formUtils";
+import { getLanguage } from "@common/constants";
+import { getLocale } from "@common/helpers/localize";
+import { notEmpty } from "@common/helpers/util";
+import TALENTSEARCH_APP_DIR from "../../talentSearchConstants";
+import {
+  Department,
+  Language,
+  GovEmployeeType,
+  Classification,
+  UpdateUserAsUserInput,
+  useCreateAccountMutation,
+  useGetCreateAccountFormDataQuery,
+} from "../../api/generated";
+import {
+  formValuesToSubmitData,
+  GovernmentInfoForm,
+} from "../GovernmentInfoForm/GovernmentInfoForm";
+import talentSearchRoutes from "../../talentSearchRoutes";
+
+type FormValues = Pick<
+  UpdateUserAsUserInput,
+  "firstName" | "lastName" | "email" | "preferredLang"
+> & {
+  govEmployeeYesNo: "yes" | "no";
+  govEmployeeType: GovEmployeeType | null;
+  lateralDeployBool: boolean;
+  currentClassificationGroup: string;
+  currentClassificationLevel: string;
+};
+
+export interface CreateAccountFormProps {
+  classifications: Classification[];
+  departments: Department[];
+  handleCreateAccount: (data: UpdateUserAsUserInput) => Promise<void>;
+}
+
+export const CreateAccountForm: React.FunctionComponent<
+  CreateAccountFormProps
+  // TODO: This never used error will be fixed after department is added to User model in separate issue.
+> = ({ classifications, departments, handleCreateAccount }) => {
```

Comment:
linting for unused var

Reply:
This will be fixed in #2990 but I guess I can remove for now.